modules/installing-trustyai-service.adoc

```diff
@@ -0,0 +1,23 @@
+:_module-type: CONCEPT
+
+[id='installing-trustyai-service_{context}']
+= Installing the TrustyAI service for a data science project
+
+[role='_abstract']
+
+For each data science project (namespace) that contains models for which the data scientists on your team want to monitor, you need to install an instance of the TrustyAI service. This action installs TrustyAI into your project and provides access to its features for all models deployed within that project.
```

Comment:
```suggestion
Install TrustyAI into your project and provides access to its features for all models deployed within that project so that data scientists can monitor models for bias. An instance of the TrustyAI service is required for each data science project, or namespace, that contains models that your team wants to monitor. 
```
Rearranged slightly - felt like the second sentence gave a larger overview of this topic than the first. Also modified to be less feature focused but feel free to change as you see fit if I've mucked it up 😄